packages/amplify-provider-awscloudformation/src/utils/admin-login-server.ts

```diff
@@ -38,7 +40,13 @@ export class AdminLoginServer {
 
   public async startServer(callback: () => void) {
     await this.setupRoute(callback);
-    this.server = this.app.listen(this.getPort());
+    // Need to specify hostname for WSL
+    this.server = this.app.listen(this.getPort(), this.getHost());
+    this.serverLocalhost = this.app.listen(this.getPort());
+  }
```

Comment:
Why is a second server needed?

Reply:
This is to make diff smaller. We have tests that tries to access to localhost and I did not want to replace them all with 127.0.0.1